server-mysql/Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM jboss/keycloak:latest
 
-ADD changeDatabase.xsl /opt/jboss/keycloak/
-RUN java -jar /usr/share/java/saxon.jar -s:/opt/jboss/keycloak/standalone/configuration/standalone.xml -xsl:/opt/jboss/keycloak/changeDatabase.xsl -o:/opt/jboss/keycloak/standalone/configuration/standalone.xml; java -jar /usr/share/java/saxon.jar -s:/opt/jboss/keycloak/standalone/configuration/standalone-ha.xml -xsl:/opt/jboss/keycloak/changeDatabase.xsl -o:/opt/jboss/keycloak/standalone/configuration/standalone-ha.xml; rm /opt/jboss/keycloak/changeDatabase.xsl
+ADD change-database.txt /opt/jboss/keycloak
+RUN /opt/jboss/keycloak/bin/jboss-cli.sh --file=/opt/jboss/keycloak/change-database.txt; \
+    sed -i -e 's/standalone/standalone-ha/' /opt/jboss/keycloak/change-database.txt; \
+    /opt/jboss/keycloak/bin/jboss-cli.sh --file=/opt/jboss/keycloak/change-database.txt;
+RUN rm -rf /opt/jboss/keycloak/standalone/configuration/standalone_xml_history/current/*; \
```

Comment:
It's a best practice to merge `RUN`s to avoid adding extra layers if possible. Second run will shade out files but they will still be present on fs layer from previous run. 

New docker versions have support for `COPY --from=...` for multi-stage builds but I'm not sure if DockerHub has support for this feature yet, it was only presented on DockerCon'17.

Disclaimer: I'm not a Keycloak developer.

Reply:
Thank you Konstantin, we will consider your advices,
Greetings